tests/unit_tests/dracut_tests/test_dracut_anaconda-lib.py

```diff
@@ -0,0 +1,177 @@
+#
+# Copyright 2025 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use, modify,
+# copy, or redistribute it subject to the terms and conditions of the GNU
+# General Public License v.2.  This program is distributed in the hope that it
+# will be useful, but WITHOUT ANY WARRANTY expressed or implied, including the
+# implied warranties of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.  Any Red Hat
+# trademarks that are incorporated in the source code or documentation are not
+# subject to the GNU General Public License and may only be used or replicated
+# with the express permission of Red Hat, Inc.
+
+import os
+import re
+import subprocess
+import unittest
+from collections import namedtuple
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+
+DISABLE_COMMAND_PREFIX = "disabled-command"
+
+
+SubprocessReturn = namedtuple("SubprocessReturn",
+                              ["returncode", "disabled_cmd_args", "stdout", "stderr"])
+
+
+class AnacondaLibTestCase(unittest.TestCase):
+
+    def setUp(self):
+        self._temp_dir = TemporaryDirectory()
+        self._content = ""
+
+        fake_command = os.path.join(self._temp_dir.name, "command")
```

Comment:
Do we still need the fake command if we disable it?

Reply:
Thanks a lot for finding this. I had a lot of trial and error approaches and this is a leftover. Fixed.